_maps/map_files/IceBoxStation/IceBoxStation.dmm

```diff
@@ -233152,9 +233152,9 @@ wNO
 wNO
 wNO
 wNO
-wNO
-wNO
-wNO
+wXw
```

Comment:
if you look very closely you'll notice that i have not changed any keys which means that both the turfs and areas are in perfect alignment. what you see on the MDB render is what you get.